apps/server/src/modules/user-import/uc/user-import.uc.ts

```diff
@@ -259,13 +331,21 @@ export class UserImportUc {
 		return currentUser;
 	}
 
-	private async updateUserAndAccount(
-		importUser: ImportUser,
-		school: LegacySchoolDo
-	): Promise<[User, Account] | undefined> {
+	private async updateUserAndAccount(importUser: ImportUser, school: LegacySchoolDo): Promise<void> {
+		const useWithUserLoginMigration: boolean = this.userImportFeatures.instance === 'n21';
```

Comment:
this.userImportFeatures.instance === 'n21' is could be moved into a private function then we only one usage of a magic string :/